content/concepts/consensus-network/consensus-protections.md

```diff
@@ -0,0 +1,67 @@
+
+Confirming transactions in a timely manner while maintaining [the desired properties of the network](intro-to-consensus.html#consensus-protocol-properties) is a complex challenge, and it's impossible to build a perfect system. The XRP Ledger Consensus Protocol is designed to work as well as it can in most situations, and to fail as gracefully as possible in the situations where it can't.
+
+This page describes some of the types of challenges that the XRP Ledger Consensus Protocol faces and how it handles them.
+
+## Individual Validators Misbehaving
+
+_Validators_ are servers who actively contribute to the process of deciding each new ledger version. Validators only have an influence over servers configured to trust them (including indirectly). Consensus can continue even if some validators are misbehaving, including a large variety of failure cases, such as:
+
+- Being unavailable or overloaded.
+- Being partially disconnected from the network, so their messages reach only a subset of participants without delay.
+- Intentionally behaving with intent to defraud others or halt the network.
+- Behaving maliciously as a result of pressure from outside factors, such as threats from an oppressive government.
+- Accidentally sending confusing or malformed messages due to a bug or outdated software.
+
+In general, consensus can continue without problems as long as only a small percentage (less than about 20%) of trusted validators are misbehaving at a given time. (For the exact percentages and the math behind them, see the latest [Consensus Research](consensus-research.html).)
+
+If more than about 20% of validators are unreachable or not behaving properly, the network fails to reach a consensus. During this time, new transactions can be tentatively processed, but new ledger versions cannot be validated, so those transactions' final outcomes are not certain. In this situation, it would become immediately obvious that the XRP Ledger is unhealthy, prompting intervention from human participants who can decide whether to wait, or reconfigure their set of trusted validators.
+
+The only way to confirm invalid transactions—such as those spending money that has already been spent, or otherwise breaking the rules of the network—would be if more than about 80% of trusted validators colluded to confirm those transactions. In other words, a large majority of trusted validators would have to misbehave in the exact same way. With dozens of trusted validators run by different people and businesses in different parts of the world, this would be very difficult to achieve intentionally.
```

Comment:
Is this correct? 

> The only way to confirm invalid transactions ...

For some reason, it seems like the only way to _not_ be able to confirm invalid transactions -- the only way to be fooled.

Reply:
The sentence is correct as written, but probably too dense. It describes the (extremely difficult) way to get invalid transactions confirmed: by controlling more than 80% of trusted validators.